has_dynamic_status_types.gemspec

```diff
@@ -4,24 +4,19 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'has_dynamic_status_types/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "has_dynamic_status_types"
+  spec.name          = 'has_dynamic_status_types'
   spec.version       = HasDynamicStatusTypes::VERSION
-  spec.authors       = ["Benjamin Martin"]
-  spec.email         = ["benjamin@salemaster.co.uk"]
-  spec.description   = %q{Add differenet statuses to your models}
+  spec.authors       = ['Benjamin Martin']
+  spec.email         = ['benjamin@salemaster.co.uk']
+  spec.description   = %q{Add different statuses to your models}
   spec.summary       = %q{Models can has different statuses, but different flavours of the same model the same status might mean something else .. crazy!}
```

Comment:
I think there is a mistype:
`Models can has different statuses` to `Models can have different statuses`